vivisect/qt/memory.py

```diff
@@ -117,15 +117,15 @@ def _hotkey_nav_prevva(self):
         self._selectVa(loc[0])
 
     @vq_hotkey.hotkey('viv:nav:nextpage')
-    def _hotkey_nav_nextva(self):
+    def _hotkey_nav_nextpage(self):
```

Comment:
copy pasta in pgup/pgdown support ended up overlapping the handler functions which caused up/down to fail to work.